app/modules/info/ping.py

```diff
@@ -4,6 +4,7 @@
 
 from app.utilities import logger
 
+
 ContextType = commands.Context
```

Comment:
This line is useless (and if it wasn't, should be done during the import), + Empty white space

```suggestion
```